cico_run_tests.sh

```diff
@@ -1,6 +1,40 @@
 #!/bin/bash
+# Output command before executing
```

Comment:
@alexeykazakov Done, we need to test it but I connected this repo with travisci  pipeline